src/style.css

```diff
@@ -0,0 +1,50 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(1),
+.stars--5 .stars__star:nth-child(1) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(2),
+.stars--5 .stars__star:nth-child(2) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(3),
+.stars--5 .stars__star:nth-child(3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(4),
+.stars--5 .stars__star:nth-child(4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-child(5) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
try using a different value for :nth-child, this will help you keep your styles more concise, and remove 4 last blocks of styles